scalasql/core/src/Queryable.scala

```diff
@@ -68,6 +68,14 @@ object Queryable {
     var nulls = 0
     var nonNulls = 0
 
+    def restAreNulls(columnsCount: Int): Boolean = {
+      val result = Range.inclusive(index + 1, index + columnsCount).map { i =>
```

Comment:
It seems you could do `forall` directly without doing `map` first, using the same predicate.